src/app/workflow/version-modal/version-modal.component.html

```diff
@@ -13,7 +13,7 @@
   ~    See the License for the specific language governing permissions and
   ~    limitations under the License.
   -->
-<h4 mat-dialog-title>{{ isPublic || !canWrite ? 'View' : 'Edit' }} Version Tag</h4>
+<h4 mat-dialog-title>{{ isPublic || !canWrite ? 'View' : 'Edit' }} {{ (isService$ | async) ? 'Service Version' : 'Version Tag' }}</h4>
```

Comment:
Should be "Service Version" and "Workflow Version" in my opinion

Reply:
I agree to avoid "Tag", we've been using that term in the context of tools and Docker container "tags" exclusively

Reply:
Changed to "Workflow Version"